librz/core/project.c

```diff
@@ -38,22 +39,33 @@ RZ_API RzProjectErr rz_project_save(RzCore *core, RzProject *prj, const char *fi
 }
 
 RZ_API RzProjectErr rz_project_save_file(RzCore *core, const char *file) {
+	char *tmp_file = rz_file_temp("svprj");
```

Comment:
I think it's better/safer to use `rz_file_mkstemp`. `rz_file_temp` uses `mktemp`, which is not good and probably we should deprecate this function...